### ISSUE TITLE
JAWS 2018 does not properly announce "role=dialog"

### ISSUE THREAD

**Issue by bnsilver** [FreedomScientific/standards-support#306]
See https://freedomscientific.github.io/VFO-standards-support/aria.html#index-aria-dialog

> Note (Aug 31, 2017): When tested with JAWS 18 & IE 11, dialog not announced when opened. Additionally, field names/labels shared across all input fields. JAWS 18 & FF 55 worked as expected.
> 
> This role will be announced only when a child of the dialog gets focus, and will not appear in the virtual buffer.

**Comment by stevefaulkner** [comment]
@bnsilver Closing this, please feel free to re-open if this still occurs with Latest (JAWS 2024).
